client/lib/abtest/active-tests.js

```diff
@@ -138,6 +138,7 @@ export default {
 		localeTargets: 'any',
 		allowExistingUsers: true,
 	},
+<<<<<<< HEAD
```

Comment:
You have unfixed merge conflicts.

Reply:
Now that the conflict resolving branch has been merged, I am marking this as resolved.